subprojects/language-java/src/main/java/org/gradle/api/tasks/JavaExec.java

```diff
@@ -727,6 +743,18 @@ public List<CommandLineArgumentProvider> getJvmArgumentProviders() {
         return javaExecSpec.getJvmArgumentProviders();
     }
 
+    /**
+     * Returns the list of configured extra JVM arguments.
+     *
+     * @return A property of extra JVM arguments.
+     * @since 8.1
+     */
+    @Input
+    @Incubating
+    public ListProperty<String> getJvmArguments() {
```

Comment:
I think it should be possible to now use:
```
public abstract ListProperty<String> getJvmArguments();
```

And drop the manual field.

Reply:
@big-guy If you don't insist, I would left it as a manually declared field to keep consistent with other properties in class. And in future we could refactor all properties at once in suggest manner. What do you think?

Reply:
We don't need to be consistent since this is just boilerplate to remove later, but I won't insist on you doing it now.